pkg/build/api/types.go

```diff
@@ -425,6 +429,18 @@ type SourceBuildStrategy struct {
 	ForcePull bool
 }
 
+// JenkinsPipelineStrategy holds parameters specific to a Jenkins Pipeline build.
+// This strategy is experimental.
+type JenkinsPipelineBuildStrategy struct {
+	// JenkinsfilePath is the optional path of the Jenkinsfile that will be used to configure the pipeline
+	// relative to the root of the source repository. If both JenkinsfilePath & Jenkinsfile are
+	// both not specified, this defaults to Jenkinsfile in the root of the specified git repo.
+	JenkinsfilePath string
```

Comment:
ok.  so the comment should align w/ the Dockerfilepath comment to make it clear it's relative to the contextDir:

```
	// DockerfilePath is the path of the Dockerfile that will be used to build the Docker image,
	// relative to the root of the context (contextDir).
```